tests/parametric/test_otel_span_methods.py

```diff
@@ -755,6 +756,121 @@ def test_otel_span_extended_reserved_attributes_overrides_analytics_event(
+                span.record_exception(message="woof", attributes={"exception.stacktrace": "stacktrace string"})
+                span.end_span()
+
+        root_span = get_span(test_agent)
+        assert "error" not in root_span or root_span["error"] == 0
+
+    @missing_feature(context.library == "golang", reason="Not implemented")
+    @missing_feature(context.library == "php", reason="Not implemented")
+    @missing_feature(context.library == "java", reason="Not implemented")
+    @missing_feature(context.library == "ruby", reason="Not implemented")
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @missing_feature(context.library <= "dotnet@2.52.0", reason="Implemented in 2.53.0")
+    @missing_feature(context.library < "python@2.10.0", reason="Implemented in 2.10.0")
+    def test_otel_record_exception_meta_serialization(self, test_agent, test_library):
+        """
+            Tests the Span.RecordException API (requires Span.AddEvent API support)
+            and its serialization into the Datadog error tags and the 'events' tag
+        """
+        with test_library:
+            with test_library.otel_start_span("operation") as span:
+                span.set_status(OTEL_ERROR_CODE, "error_desc")
+                span.record_exception(message="woof1", attributes={"string_val": "value", "exception.stacktrace": "stacktrace1"})
+                span.add_event(name="non_exception_event", attributes={"exception.stacktrace": "non-error"})
+                span.record_exception(message="woof3", attributes={"exception.message": "message override"})
+                span.end_span()
+
+        root_span = get_span(test_agent)
+        assert "events" in root_span["meta"]
+
+        events = json.loads(root_span.get("meta", {}).get("events"))
+        assert len(events) == 3
+
+        event1 = events[0]
+        assert event1.get("name") == "exception"
+        assert event1["attributes"].get("string_val") == "value"
+        assert event1["attributes"].get("exception.message") == "woof1"
+        assert event1["attributes"].get("exception.stacktrace") == "stacktrace1"
+        assert event1.get("time_unix_nano") > 0
+
+        event2 = events[1]
+        assert event2.get("name") == "non_exception_event"
+        assert event2["attributes"].get("exception.stacktrace") == "non-error"
+        assert event2.get("time_unix_nano") > event1.get("time_unix_nano")
+
+        event3 = events[2]
+        assert event3.get("name") == "exception"
+        assert event3["attributes"].get("exception.message") == "message override"
+        assert event3.get("time_unix_nano") > event2.get("time_unix_nano")
+
+        assert root_span["error"] == 1
+        error_message = root_span["meta"].get("error.message") or root_span["meta"].get("error.msg")
+        assert error_message == "message override"
+        assert "error.type" not in root_span["meta"]
+        assert "error.stack" not in root_span["meta"]
```

Comment:
To maintain compatibility with error tracking should we always attempt set these span tags in the records exception API? 

```suggestion
        assert "error.type" in root_span["meta"]
        assert "error.stack" in root_span["meta"]
```

Reply:
In python:

root_span["meta"]["error.type"] == 'Exception'
root_span["meta"]["error.stack"] == 'Exception: woof3', this stacktrace captures the initial error.message 🤔

Reply:
Yes I think we should try to have `error.type` and `error.stack` be set so your suggestion makes sense. I'm not sure if all of the implementing languages will be able to pass this, so I could see us removing these lines if that turns out to be the case